idd/Energy+.idd.in

```diff
@@ -31865,6 +31865,7 @@ ZoneHVAC:FourPipeFanCoil,
         \key CyclingFan
         \key VariableFanVariableFlow
         \key VariableFanConstantFlow
+        \key MultiStageFan
```

Comment:
@Nigusse That's interesting.  If that's a more common term in the field, the I won't argue.  Perhaps you could double-check with the original requestor.  Either way it needs explanation to distinguish it from the existing CyclingFan option.